.buildkite/pipelines/intake.template.yml

```diff
@@ -0,0 +1,57 @@
+steps:
```

Comment:
We currently also trigger a snapshot DRA build at the end of every success intake job. Should we continue to do that?

Reply:
Oops, the DRA pipeline wasn't ready yet originally and I forgot to go back and add it. It's added now.